src/components/Icons/iconVariables.ts

```diff
@@ -0,0 +1,110 @@
+export const iconAlignArray = ["left", "right", "none"] as const;
```

Comment:
I've noticed that you now are using `as const` on some export statements.  Why is that?